tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -666,6 +687,70 @@ void ordering_test_cases() {
         ordered_containers_test(a1, a2, b1);
         ordered_iterator_test(a1.begin(), a1.begin(), a1.end(), a1.cbegin(), a1.cbegin(), a1.cend());
     }
+    { // allocator

```

Comment:
I don't think we need any test coverage here. The comparison expressions previously implemented by the removed functions are tested elsewhere, the fact that those test keep passing is proof enough without adding redundant coverage.

Reply:
Many of the other test categories in this file do that redundant coverage on equality. Should we remove them too or just apply it to everything in this PR and anything else moving forward?

Not all of these test the actual test computation of the result, just it's existence/callability (i.e. scoped_allocator_adaptor). Given how trivial the impls are that is probably ok though.

Reply:
Blech. Let's leave this as-is and I'll cleanup any doubled coverage in a followup after we're all done touching this test.